app/src/main/java/com/example/dine_and_donate/Models/Event.java

```diff
@@ -1,17 +1,33 @@
 package com.example.dine_and_donate.Models;
 
 
+import com.google.firebase.database.IgnoreExtraProperties;
+
 import java.io.File;
 import java.util.ArrayList;
 import java.util.Date;
 
+@IgnoreExtraProperties
 public class Event {
-    private String name;
-    private String info;
-    private Double percentOfMealPriceDonated;
-    private ArrayList<Date> eventDates;
-    private String voucherUrl;
-    private ArrayList<Consumer> consumers;
-    private Organization organization;
-    // could create restaurant class and include here
+    public String orgId;
+    public double locationLong;
+    public double locationLat;
+    public String locationString;
+    public long startTime;
+    public long endTime;
+    public String info;
+
+    public Event() {
```

Comment:
is this always gonna be empty? if so you can just make it one line - 

public Event() {}